client/lib.go

```diff
@@ -26,15 +26,15 @@ const (
 
 func NewGoShimmerAPI(node string, httpClient ...http.Client) *GoShimmerAPI {
 	if len(httpClient) > 0 {
-		return &GoShimmerAPI{node: node, httpClient: httpClient[0]}
+		return &GoShimmerAPI{baseUrl: node, httpClient: httpClient[0]}
```

Comment:
Make `NewGoShimmerAPI` parameter name consistent with the struct field. It should either both be node or baseUrl